md/guest-user.md

```diff
@@ -64,31 +64,45 @@ To activate it:
 
 1.  Use the [platform setup tool](BonitaBPM_platform_setup) to retrieve the current configuration (Eg. setup.sh/.bat pull). You need to execute the following actions in the folder of the tenant in which the applications which requires public access are deployed.
 
 2. In the tenant_portal folder of the target tenant: `<BUNDLE_HOME>/setup/platform_conf/current/tenants/<TENANT_ID>/tenant_portal`,
-   update the file `custom-permissions-mapping.properties` to add the following line:
+   update the file `dynamic-permissions-checks-custom.properties` to uncomment 3 lines as follows (they may already be uncommented if you use the dynamic authorisation checking instead of the default static authorization checking) :
     ```
-            profile|<guest_profile_name>=[case_start] 
+        [...]
+        ## ProcessPermissionRule
+        ## Let the user do get only on processes he deployed or that he supervised
+    --> GET|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #POST|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #PUT|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #DELETE|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+    --> GET|bpm/process/*/instantiation=[profile|Administrator, check|org.bonitasoft.permissions.ProcessInstantiationPermissionRule]
+    --> POST|bpm/process/*/instantiation=[profile|Administrator, check|org.bonitasoft.permissions.ProcessInstantiationPermissionRule]
+        [...]
     ```
-    or
-    ```
-            user|<guest_username>=[case_start] 
-    ```
-    Make sure to replace <guest_profile_name> or <guest_username> by the name of the guest profile or guest user account created in the section [Guest profile and dedicated user account](#prerequisite).
-    
 :::info 
-**Note:** If you use the [dynamic authorisation checking](rest-api-authorization#dynamic_authorization) instead of the default static authorization checking, this modification is not necessary as the rule `ProcessInstantiationPermissionRule` should already grant the access to the guest account when it is mapped to the actor initiator of the process.
+**Note:** This modification will activate the [dynamic authorisation checking](rest-api-authorization#dynamic_authorization) instead of the default static authorization checking for the following request : `GET|bpm/process`, `GET|bpm/process/*/instantiation`, `POST|bpm/process/*/instantiation`. It does not impact only the guest user access to these resources but the whole organization.  
+When a user tries to get the contract or general information of a process it will make sure he is either:
```

Comment:
What about access to the context of a case?

Reply:
The context of a case if not impacted by this change. This is a different API call so different resource with different permissions